src/stage1/codegen.cpp

```diff
@@ -5197,6 +5199,7 @@ static LLVMValueRef get_int_builtin_fn(CodeGen *g, ZigType *expr_type, BuiltinFn
         n_args = 1;
         key.id = ZigLLVMFnIdBitReverse;
         key.data.bit_reverse.bit_count = (uint32_t)int_type->data.integral.bit_count;
+        key.data.bit_reverse.vector_len = vector_len;
```

Comment:
```cpp
        struct {
            uint32_t bit_count;
        } bit_reverse;
```
in all_types.hpp needs to be changed or this removed. I guess removing this should work.